Sources/SwifterSwift/UIKit/UIViewExtensions.swift

```diff
@@ -472,6 +480,45 @@ public extension UIView {
         CATransaction.commit()
     }
 
+    /// SwifterSwift: Apply Gradient Colors.
+    ///
+    ///     view.applyGradient(
+    ///         colors: [UIColor.red.cgColor, UIColor.blue.cgColor],
+    ///         locations: [0.0, 1.0],
+    ///         direction: .topToBottom
+    ///     )
+    ///
+    /// - Parameters:
+    ///   - colors: An array of colors defining the color of each gradient stop.
+    ///   - locations: An array of NSNumber defining the location of each
+    ///                gradient stop as a value in the range [0,1]. The values must be
+    ///                monotonically increasing.
+    ///   - direction: Enumeration type describing the direction of the gradient.
+    func applyGradient(colors: [Any]?, locations: [NSNumber]? = [0.0, 1.0], direction: GradientDirection = .topToBottom) {
```

Comment:
Just because the old CoreAnimation code doesn't have a great API doesn't mean that we have to do the same.
```suggestion
    func applyGradient(colors: [Color]?, locations: [CGFloat]? = [0.0, 1.0], direction: GradientDirection = .topToBottom) {
```
See suggestions below to conform to this new interface.

Explanation: Using `Color` instead of `Any` for `colors` should be pretty self-explanatory. It makes calls to the function cleaner and easier, and removes chances of user error. Using `CGFloat` instead of `NSNumber`, however, is less important because calling the function stays the same, however it's more UIKit-esque, in my opinion, and removes strange calls such as `locations: [NSNumber(value: false)]`, which would be allowed but conceptually illogical.
It would also make it consistent with the style used in `CAGradientLayerExtensions.swift`.